pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -197,6 +207,15 @@ def transpose(input: Tensor, dim0: int, dim1: int):
   return tensorflow_transpose(input, perm={dim0: dim1, dim1: dim0})
 
 
+def t(input: Tensor):
+  if len(input.shape) < 2:
+    return input
+  elif len(input.shape) == 2:
+    return transpose(input, 0, 1)
+  else:
+    raise ValueError("t() expects input to be <= 2-D tensor")
```

Comment:
Is that the case? Or not implemented?

Reply:
Yes, that's stated in the PyTorch docs https://pytorch.org/docs/stable/generated/torch.t.html#torch.t

Reply:
Ok. Maybe add a small comment. This is a bit unintuitive (because NumyPy and TensorFlow allow any N-D I think).